packages/core/src/button/base-button.element.scss

```diff
@@ -168,13 +168,19 @@
   --border-color: var(--color);
 }
 
-:host([action='flat']) {
+:host([action*='flat']) {
   --background: #{$cds-alias-object-opacity-0};
   --border-color: #{$cds-alias-object-opacity-0};
   --color: #{$cds-alias-status-info};
   --box-shadow-color: #{$cds-alias-object-opacity-0};
 }
 
+:host([action='flat-inline']) {
+  --height: auto;
+  --padding: 0;
```

Comment:
I thought we only wanted to remove padding from the left and the right. Why would we remove all padding here?

Reply:
The buttons in the card footer are adding additional bottom space and it is not possible to remove that space without an additional token. Like we do with `  --card-remove-margin`. 

`--height: auto` will help us accomplish to remove that space and have an even padding that won't make the footer bigger than what's needed.

Also I and Cory discuss that and we thought having a `--height: auto` will bring more `inline` feeling.